### DISCUSSION TITLE
[docusaurus] Some problems I'm facing while setting up the extension

### DISCUSSION THREAD

**Discussion by Losses** [tgreyuk/typedoc-plugin-markdown#339] [Help] [answered]
Hi! Thanks for your great plugin, it helped us a lot!

But I'm facing a problem while setting up the extension:

## Functions are not displayed in the sidebar:

Functions are listed in the "modules" page but not listed in the sidebar, this looks strange, since classes and classes and interfaces are available in the sidebar, I searched the source code but didn't find any useful cue...

## Some functions are exported

Some definitions of exported functions are not shown on the document.

## The modules page is not listing all exported things

Actually from the official example, the `/modules` page listed all things available in the package:

https://typedoc.org/example/modules.html

But the modules page generated by this extension only listed limited content with details but not a list off all items, it confused me, is this a on purposed design? Is it possible to generate a page looks like the official example?

Thank you for your time for reading my question!

**Comment by iway1** [comment]
i legit can't figure out how this library is supposed to work or what lol I'm just like manually manipulating it's output at this point 🤣

**Comment by tgreyuk** [comment, marked as answer]
https://github.com/tgreyuk/typedoc-plugin-markdown/discussions/376
